Makefile

```diff
@@ -178,12 +181,11 @@ release-localkube: out/localkube checksum
 update-releases: 
 	gsutil cp deploy/minikube/k8s_releases.json gs://minikube/k8s_releases.json
 
-out/localkube-image: out/localkube
-	# TODO(aprindle) make addons placed into container configurable
-	docker build -t $(REGISTRY)/localkube-image:$(TAG) -f deploy/docker/Dockerfile .
-	@echo ""
-	@echo "${REGISTRY}/localkube-image:$(TAG) succesfully built"
-	@echo "See https://github.com/kubernetes/minikube/tree/master/deploy/docker for instrucions on how to run image"
+localkube-image: .localkube-image
+.localkube-image: out/localkube deploy/docker/Dockerfile $(ADDON_FILES)
```

Comment:
I don't really follow what the ".localkube-image" target is for. Any reason to not just make this a PHONY target?